parcels/scripts/plottrajectoriesfile.py

```diff
@@ -13,7 +13,7 @@
 try:
     if sys.platform == 'darwin' and sys.version_info[0] == 3:
         import matplotlib
-        matplotlib.use("TkAgg")
+        matplotlib.use("Agg")
```

Comment:
hm - weird that TkAgg doesn't work - it's (for me for auto-generating the notebooks) actually the only thing that works consistently. Anyway, if this works - nice!